README.md

```diff
@@ -74,10 +88,33 @@ This will compile and store all Lua files in the given folder including subfolde
 #### Output
 Depending on what type(s) of firmware you built this will create one or two LFS images in the root of your lua folder.
 
-#### Options
-Same as for ``build`` command.
+## Build for ESP32
+
+NodeMCU for ESP32 is built on the [ESP-IDF](https://github.com/espressif/esp-idf) (Espressif IoT Development Framework). It uses a menu-driven user interface Kconfig to configure all firmware features and options. Hence, building NodeMCU for ESP32 is a two step process and you will launch the Docker container twice. First to start Kconfig, select all options and write the configuration file. Then to actually build the firmware.
+
+### Configure modules and features
+
+``docker run --rm -ti -v `pwd`:/opt/nodemcu-firmware marcelstoer/nodemcu-build configure-esp32``
 
-### Note for Windows users
+This internally will run `make menuconfig` in the firmware directory. It will generate a `sdkconfig` file in the same.
+
+### Build the firmware
+
+``docker run --rm -ti -v `pwd`:/opt/nodemcu-firmware marcelstoer/nodemcu-build build``
+
+That is the exact same command as for building for the ESP8266. It uses Git to figure out whether you checked out NodeMCU for ESP32 or ESP8266. The `build` command is thus a shortcut to using `build-esp32`.
+
+The process will fairly early with a meaningful error message if it does not find a `sdkconfig` file in the firmware directory.
```

Comment:
should be "fail"

Reply:
Thanks for the diligent review; I fixed that half an hour after the initial commit.